doc/_includes/footer.html

```diff
@@ -23,8 +23,7 @@
     <!-- FOOTER  -->
     <div id="footer_wrap" class="outer">
       <footer class="inner">
-        <p class="copyright">Cap'n Proto is a project of <a href="https://sandstorm.io">Sandstorm.io</a></p>
-        <p>Maintained by <a href="https://github.com/kentonv">kentonv</a> ∙ Design by <a href="http://www.starfruit-cafe.net/blog">sailorhg</a>
+        <p>Maintained by <a href="https://github.com/kentonv">kentonv</a> and friends ∙ Design by <a href="http://www.starfruit-cafe.net/blog">sailorhg</a>
```

Comment:
While you're here, sailorhg's domain appears to be out of date. It currently redirects to a stock/crypto trading platform.

Reply:
Oh gross. Good catch, fixed.